go.mod

```diff
@@ -45,6 +45,7 @@ require (
 	github.com/spf13/afero v1.2.1 // indirect
 	github.com/spf13/cobra v0.0.3
 	github.com/spf13/pflag v1.0.3 // indirect
+	github.com/stretchr/testify v1.3.0
```

Comment:
I would prefer we stick with the gomega package since thats the framework used by Kubebuilder.

Reply:
Gomega looks very ugly :( And `testify` is a slim set of matchers. Do we really, really have to?

Reply:
I'm ok with keeping testify and gomega. I've typically used ginkgo+gomega for e2e tests and just the built-in test package (without other matchers) for unit.